src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int numberOfBall = 100;

```

Comment:
This constant should be private, remember about encapsulation. Constant names use CONSTANT_CASE ([read this article ](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names))